roles/installer/templates/configmaps/redirect-page.configmap.html.j2

```diff
@@ -66,7 +66,7 @@ data:
         <p class="doc-note">
             The API endpoints for this platform service will temporarily remain available at the URL for this service.
             Please use the Ansible Automation Platform API endpoints corresponding to this component in the future.
-            These can be found at <a href="{{ public_base_url }}/api/{{ deployment_type }}" target="_blank">{{ public_base_url }}/api/{{ deployment_type }}</a>.
+            These can be found at <a href="{{ public_base_url }}/api/{{ deployment_type_shortname }}" target="_blank">{{ public_base_url }}/api/{{ deployment_type_shortname }}</a>.
```

Comment:
@jamesmarshall24 I just tested this out and without the trailing slash, it's a 404.  That's technically an nginx conf bug, but can you add the slash on the end, which should make it work until we can fix the nginx.conf.

```suggestion
            These can be found at <a href="{{ public_base_url }}/api/{{ deployment_type_shortname }}" target="_blank">{{ public_base_url }}/api/{{ deployment_type_shortname }}/</a>.
```